fleetactivitytracking/views.py

```diff
@@ -87,6 +105,43 @@ def fatlink_view(request):
 
     return render(request, 'fleetactivitytracking/fatlinkview.html', context=context)
 
+@login_required
+@permission_required('auth.fleetactivitytracking_statistics')
+def fatlink_statistics_corp_view(request, corpid, year=None, month=None):
+    if year is None:
+        year = datetime.date.today().year
+    if month is None:
+        month = datetime.date.today().month
+
+    year = int(year)
+    month = int(month)
+    start_of_month = datetime.datetime(year, month, 1)
+    start_of_next_month = first_day_of_next_month(year, month)
+    start_of_previous_month = first_day_of_previous_month(year, month)
+    context = {}
+    fat_stats = {}
+    corp_members = EveCharacter.objects.filter(corporation_id=corpid).values('user_id').distinct()
+
+    for member in corp_members:
+        try:
+            fat_stats[member['user_id']] = MemberStat(member, start_of_month, start_of_next_month)
+        except ObjectDoesNotExist:
+            continue
+
+    # collect and sort stats
+    stat_list = [fat_stats[x] for x in fat_stats]
+    stat_list.sort(key=lambda stat: stat.mainchar.character_name)
+    stat_list.sort(key=lambda stat: (stat.n_fats, stat.n_fats / stat.n_chars), reverse=True)
+
+    if datetime.datetime.now() > start_of_next_month:
+        context = {'fatStats': stat_list, 'month': start_of_month.strftime("%B"), 'year': year,
```

Comment:
Might make more sense to set this outside of the conditional statement and only add `next_month` inside your if statement.